hack/builder/Dockerfile

```diff
@@ -40,8 +29,16 @@ RUN curl -fsSL $LIBNL_DOWNLOAD_URL -o libnl.tar.gz \
 RUN go get -u github.com/kardianos/govendor
 RUN go get -u golang.org/x/tools/cmd/goimports
 RUN go get -u github.com/mitchellh/gox
-RUN go get -u github.com/alecthomas/gometalinter
-RUN gometalinter --install
+
+# gometalinter 2.0.11:
```

Comment:
If we tell people to run hack/install-metalinter, is there a reason we don't do that ourselves?

Reply:
I considered it, but I got a little nervous not checksumming *something* that we download.  The gometalinter-provided install script downloads both the tarball and the checksum file from the same place, and then does the check.  If that's secure enough, then I'm game to make the change.

Reply:
I removed `hack/install-metalinter.sh`, by the way - the instructions now point at the one in the gometalinter repo.